### PR TITLE
Adjust addons components variables、resources, ...etc

### PR DESCRIPTION
**Pull request by wangzhuzhen**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: erda-helm/erda/charts/cassandra/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+description: A Helm chart for deploy cassandra
 type: application
-version: 0.1.0
\ No newline at end of file
+version: 3.11.10
+keywords:
+  - cassandra
+  - operator
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/cassandra/Chart.yaml:10]: delete it.

File: erda-helm/erda/charts/elasticsearch/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: elasticsearch
 description: A Helm chart for elasticsearch
 type: application
-version: 0.1.0
+version: 6.2.4
+keywords:
+  - elasticsearch
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/elasticsearch/Chart.yaml:9]: delete it.

File: erda-helm/erda/charts/elasticsearch/templates/statefulset.yaml [hunk 3]
```diff
@@ -75,12 +75,27 @@ spec:
...
+                memory: {{ .Values.resources.limits.memory | default "8Gi" | quote }}
+            {{- end }}
+            requests:
+              cpu: {{ .Values.resources.requests.cpu | default "200m" | quote }}
+              memory: {{ .Values.resources.requests.memory | default "819Mi" | quote }}
             limits:
-              cpu: {{ .Values.elasticsearch.resources.limits.cpu |  default "2" | quote }}
-              memory: {{ .Values.elasticsearch.resources.limits.memory | default "8Gi" | quote }}
+              cpu: {{ .Values.resources.limits.cpu |  default "2" | quote }}
+              memory: {{ .Values.resources.limits.memory | default "8Gi" | quote }}
```
Comment (luobily) [inline erda-helm/erda/charts/elasticsearch/templates/statefulset.yaml:98]: Two more lines?

File: erda-helm/erda/charts/etcd/Chart.yaml [hunk 4]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: etcd
 description: A Helm chart for etcd
 type: application
-version: 0.1.0
\ No newline at end of file
+version: 3.3.15-0
+keywords:
+  - etcd
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/etcd/Chart.yaml:9]: delete it.

File: erda-helm/erda/charts/etcd/templates/etcd-cluster.yaml [hunk 5]
```diff
@@ -129,33 +131,33 @@ spec:
...
       tolerations:
         - operator: Exists
       containers:
-        - name: {{ .Values.etcd.name | default "erda-etcd" }}-1
-          image: {{ .Values.global.image.repository}}/erda-etcd:{{ .Values.etcd.version }}
+        - name: erda-etcd-1
+          image: {{ .Values.global.image.repository}}/erda-etcd:{{ .Values.tag }}
           imagePullPolicy: IfNotPresent
           env:
             - name: PODIP
```
Comment (luobily) [inline erda-helm/erda/charts/etcd/templates/etcd-cluster.yaml:163]:
```suggestion
           - name: POD_IP
            valueFrom:
              fieldRef:
                apiVersion: v1
                fieldPath: status.podIP
          - name: HOST_IP
            valueFrom:
              fieldRef:
                apiVersion: v1
                fieldPath: status.hostIP
```

File: erda-helm/erda/charts/etcd/templates/etcd-cluster.yaml [hunk 6]
```diff
@@ -242,33 +245,33 @@ spec:
...
       tolerations:
         - operator: Exists
       containers:
-        - name: {{ .Values.etcd.name | default "erda-etcd" }}-2
-          image: {{ .Values.global.image.repository}}/erda-etcd:{{ .Values.etcd.version }}
+        - name: erda-etcd-2
+          image: {{ .Values.global.image.repository}}/erda-etcd:{{ .Values.tag }}
           imagePullPolicy: IfNotPresent
           env:
             - name: PODIP
```
Comment (luobily) [inline erda-helm/erda/charts/etcd/templates/etcd-cluster.yaml:277]:
```suggestion
            - name: POD_IP
            valueFrom:
              fieldRef:
                apiVersion: v1
                fieldPath: status.podIP
          - name: HOST_IP
            valueFrom:
              fieldRef:
                apiVersion: v1
                fieldPath: status.hostIP
```

File: erda-helm/erda/charts/kafka/Chart.yaml [hunk 7]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: kafka
 description: A Helm chart for kafka
 type: application
-version: 0.1.0
\ No newline at end of file
+version: 1.1.0-20210323-be01a9b
+keywords:
+  - kafka
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/kafka/Chart.yaml:9]: delete it.

File: erda-helm/erda/charts/kms/Chart.yaml [hunk 8]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: kms
 description: A Helm chart for kms
 type: application
-version: 0.1.0
+version: 1.0.0-20200608-f11445f
+keywords:
+  - kms
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/kms/Chart.yaml:9]: delete it.

File: erda-helm/erda/charts/redis/Chart.yaml [hunk 9]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: redis
 description: A Helm chart for redis
 type: application
-version: 0.1.0
\ No newline at end of file
+version: 2.7.1
+keywords:
+  - registry
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/redis/Chart.yaml:9]: delete it

File: erda-helm/erda/charts/registry/Chart.yaml [hunk 10]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: registry
 description: A Helm chart for registry
 type: application
-version: 0.1.0
+version: 2.7.1
+keywords:
+  - registry
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/registry/Chart.yaml:9]: delete it.

File: erda-helm/erda/charts/sonar/Chart.yaml [hunk 11]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: sonar
 description: A Helm chart for sonar
 type: application
-version: 0.1.0
+version: 8.4.2
+keywords:
+  - sonar
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/sonar/Chart.yaml:9]: delete it.

File: erda-helm/erda/charts/volume-provisioner/Chart.yaml [hunk 12]
```diff
@@ -2,4 +2,14 @@ apiVersion: v2
...
 type: application
-version: 0.1.0
\ No newline at end of file
+version: 1.0.0-20210416-1a3758f
+keywords:
+  - volume-provisioner
+  - csi
+  - storageclass
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/volume-provisioner/Chart.yaml:11]: delete it.

File: erda-helm/erda/charts/volume-provisioner/templates/volume-provisioner-damonset.yaml [hunk 13]
```diff
@@ -2,22 +2,22 @@ kind: DaemonSet
...
   name: erda-volume-provisioner
   namespace: {{ .Values.global.namespace | default "erda-system" }}
 spec:
   selector:
     matchLabels:
-      app: erda-volume-provisioner
+      app: volume-provisioner
   updateStrategy:
-    type: {{ .Values.global.updateStrategy.type | default "RollingUpdate" }}
+    type: RollingUpdate
```
Comment (luobily) [inline erda-helm/erda/charts/volume-provisioner/templates/volume-provisioner-damonset.yaml:13]: Need to be configurable.
```suggestion
    type: {{ .Values.global.updateStrategy.type | default "RollingUpdate" }}
```

File: erda-helm/erda/charts/zookeeper/Chart.yaml [hunk 14]
```diff
@@ -2,4 +2,12 @@ apiVersion: v2
 name: zookeeper
 description: A Helm chart for zookeeper
 type: application
-version: 0.1.0
\ No newline at end of file
+version: 3.4.13-monitor
+keywords:
+  - zookeeper
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/zookeeper/Chart.yaml:9]: delete it.

**Comment by wangzhuzhen** [comment]
do it in another pr: #30

**Pull request closed without merging**